src/lib.rs

```diff
@@ -1,3 +1,4 @@
+#![feature(crate_visibility_modifier)]
```

Comment:
Is it really necessary? Pearl will not compile on stable rust with it.

Reply:
Removed